tests/installation/installer_desktopselection.pm

```diff
@@ -29,11 +29,11 @@ sub run {
     }
     if (get_var('NEW_DESKTOP_SELECTION')) {
         # select computer role
-        if ($d ne 'kde' && $d ne 'gnome' && $d ne 'textmode') {
+        if ($d ne 'kde' && $d ne 'gnome' && $d ne 'textmode' && $d ne 'serverro') {
```

Comment:
`$d !~ /kde|gnome|textmode|serverro/` should work if you find this more readable. I do :)